docs/index.rst

```diff
@@ -184,7 +184,8 @@ Os-release file
 ===============
 
 The os-release file is looked up using the path name ``/etc/os-release``. Its
-optional additional location ``/usr/lib/os-release`` is ignored.
+optional additional location ``/usr/lib/os-release`` would be looked up for if
+the former couldn't be read.
```

Comment:
I see one more place in `index.rst` that _may_ need an extension like that. It's in section `Compatibility` and it reads:

> * The content of the `/etc/os-release` file, if any.